.github/workflows/unit-test-on-pull-request.yml

```diff
@@ -148,8 +148,8 @@ jobs:
       - name: Install dependencies
         run: |
           case "${{ matrix.target_arch }}" in
-            amd64) sudo apt-get -y install qemu-system-x86;;
```

Comment:
Should this address the issues we see currently - like https://github.com/open-telemetry/opentelemetry-ebpf-profiler/actions/runs/13582862442/job/37972057643#step:3:168? If not, which issues should these address?

As the current issues are likely routing or DNS issues (`404  Not Found [IP: 52.252.75.106 80]`), I don't think adding `apt-get update -y` will help here.

Reply:
It also helps with other possible failures, e.g. when package versions are no longer available but are still in the database.

Reply:
the magic of CI and CDN ...

Reply:
Well, you can experience the same locally when updating Debian or Ubuntu.
Better first `apt-get update` before `apt-get upgrade` or `apt-get dist-upgrade`.